plugins/ca/root-ca/src/test/java/org/apache/cloudstack/ca/provider/RootCAProviderTest.java

```diff
@@ -133,17 +128,17 @@ public void testRevokeCertificate() throws Exception {
 
     @Test
     public void testCreateSSLEngineWithoutAuthStrictness() throws Exception {
-        overrideDefaultConfigValue(RootCAProvider.rootCAAuthStrictness, "_defaultValue", "false");
+        provider.rootCAAuthStrictness = Mockito.mock(ConfigKey.class);
+        Mockito.when(provider.rootCAAuthStrictness.value()).thenReturn(Boolean.FALSE);
         final SSLEngine e = provider.createSSLEngine(SSLUtils.getSSLContext(), "/1.2.3.4:5678", null);
-        Assert.assertFalse(e.getUseClientMode());
```

Comment:
That's essentially removing the test case, I would actually disable or remove the test @GutoVeronezi

Reply:
@rhtyd actually if you take a look at this file in [4.15](https://github.com/apache/cloudstack/blob/4.15/plugins/ca/root-ca/src/test/java/org/apache/cloudstack/ca/provider/RootCAProviderTest.java), you will notice that this line is already removed, I made the changes using it as a reference.   
My PR does not change the test case, it just change the way that variable is mocked.